DOCS-7011-updated cached time unit (#282)
mulesoft/docs-api-manager: modules/ROOT/pages/policy-openid-connect.adoc

== Performance

> To improve performance, once a token is validated, it is cached by the system by default, until the token expires. In certain cases, you might reduce or even disable the caching mechanism, for example to take into account revoked token scenarios.

Reviewer: ```suggestion
After the system validates a token, by default the system improves performance by caching the token until it expires. In certain cases such as revoked token scenarios, you may reduce or even disable caching.
```

Author: Done! Thanks!

> To control token cached time before retrying validation against the OAuth Provider), specify the following property:

Reviewer: ```suggestion
To control token cached time before retrying validation against the OAuth Provider, specify the following property:
```

Author: Done! Thanks!

[listing]

> anypoint.platform.max_federation_expiration_time=<numeric value indicating time in seconds>

Reviewer: Should there be a sentence saying something like "To disable caching, set the value to zero." ?

Author: Done! Thanks!
